src/main/java/appeng/parts/reporting/AbstractPartMonitor.java

```diff
@@ -79,6 +79,7 @@
 {
 	private static final IWideReadableNumberConverter NUMBER_CONVERTER = ReadableNumberConverter.INSTANCE;
 	private IAEItemStack configuredItem;
+	private String itemStackSize;
```

Comment:
what about `lastTextDisplay` or just `lastText` and add a comment on top of the variable. Else this just sounds like a HÄÄÄ moment few months later.